spec/rails_helper.rb

```diff
@@ -54,4 +58,21 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    # Choose a test framework:
+    with.test_framework :rspec
+    # with.test_framework :minitest
```

Comment:
@isabloCh you should remove this 3 lines:

```
# with.test_framework :minitest
# with.test_framework :minitest_4
# with.test_framework :test_unit
```

it doesn't make sense to have it since you use rspec